pages/booklist.html

```diff
@@ -1,23 +1,41 @@
 <!DOCTYPE html>
 <html>
 <head>
-<title>BookList</title>
+<title>Mount Sion CBS Secondary School</title>
     
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.0/semantic.min.css">
- <link src="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.0/semantic.min.js"></link>
-  <script src="//code.jquery.com/jquery-2.2.1.js"></script>
+    <link rel="stylesheet" href="style.css">
+ <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.0/semantic.min.css">
+  <link src="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.0/semantic.min.js"></link>
+   <script src="//code.jquery.com/jquery-2.2.1.js"></script>
     
 </head>
  <body>
     
-<ul>
-  <li><a class="active" href="about.html">About</a></li>
-  <li><a href="index.html">Home</a></li>
-  <li><a href="history.html">History</a></li>
-  <li><a href="contact.html">Contact</a></li>
-</ul> 
-    
-    
-    
+<div class="ui inverted tabular menu">
```

Comment:
Do we want the menu to be responsive to different screen sizes? If so add the keyword `stackable`